src/migrations/003_oauth-access-token.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  const result = await knex('changelog').select('id').where({id: 3});
+
+  if (result.length) {
+    // Migration has been applied using the old patch system
+    return;
+  }
+  await knex('changelog').insert({
```

Comment:
We've done away with transactions here, is this intentional?

Reply:
Knex wraps these migrations in something that basically works the same. The transactions in the raw sql files were helpful for certain edge cases (specifically until the fist CREATE/ALTER table that has a syntax error, but not after). I believe knex works the same.